src/server/api/routes/users/v2/index.js

```diff
@@ -1,12 +1,14 @@
 const { Router } = require('express');
 
 const auth = require('./auth');
+const email = require('./email');
 const follow = require('./follow');
 const nfts = require('./nfts');
 
 const router = Router();
 
 router.use(auth);
+router.use(email);
```

Comment:
Do you mean maybe we can add a `userInfo` route to handle data that might be edited on the same page? Like `email`, `displayName` etc.?

Reply:
email seems somewhere in between, since it's mostly just a get set of private user data, but there is also actual email verification logics
can be it as individual route if no conclusion